adapter/noop2/noop_test.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2017 Istio Authors.
```

Comment:
If we auto generate noop, we also autogenerate _test for it ? It seems we should and we might be able to, looking at the content of the test below.